src/codeu/chat/common/NetworkCode.java

```diff
@@ -31,6 +31,7 @@
       NEW_USER_RESPONSE = 12,
       NEW_CONVERSATION_REQUEST = 13,
       NEW_CONVERSATION_RESPONSE = 14,
+      REMOVE_CONVERSATION_REQUEST = 15,
```

Comment:
Okay, that makes sense. I added it in. Let me know if I need to anything else in.